src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/main-content/MainContent.scss

```diff
@@ -0,0 +1,5 @@
+.mainContent {
+  & > div > div {
```

Comment:
`& .tableRow` ?

Reply:
It won't work as the .tableRow class comes from the `TranscriptsTable`

Reply:
What happens if you try to import classes from one file to the other with `@use`? ([docs](https://sass-lang.com/documentation/at-rules/use))

I see SASS [strongly discourages](https://sass-lang.com/documentation/at-rules/import) the use of `@import` directive, and we are using it in our code all the time

Reply:
I tried to use `@use` with mixins for this, but webpack keeps complaining about this. I will investigate this further and see if it can be fixed.

Reply:
Oh, what I didn't realise was that Sass regards imported files as partials (which have to start with an underscore). Which means, I guess, that we can't use the `@use` directive in this case.

I think in this particular case I would prefer all styles to be defined in a single style file then. Less desirable (but still tolerable) would be the `[class*='tableRow']` trick. `& > div > div` is just super fragile!

Reply:
Sorry, I was actually being stupid in both my initial and subsequent suggestion. There is a very simple solution to the problem we are discussing, and we’ve used it tons of times before :smiley: 

in the tsx file:
```javascript
import classNames from 'classnames';

import styles from './MainContent.scss';
import tableStyles from '../TranscriptsTable.scss';

// then in the component
  const rowClassNames = classNames(
    tableStyles.tableRow,
    styles.tableRow
  );

// and then during rendering
<div className={rowClassNames}>
   ...
</div>
```

Reply:
I tried `[class*='tableRow'] already and it worked well, though looked hacky. Using `classNames` looks better 😀

Reply:
✅